stf.build/include/top.xml

```diff
@@ -322,7 +322,7 @@ limitations under the License.
 	</target>
 
 	<!--
-		Some tests may require test.debugging requires tools.jar for building (e.g. test.debugging in openjdk-systemtests).
+		Some tests may require test.debugging requires tools.jar for building (e.g. test.debugging in aqa-systemtests).
```

Comment:
predates this change, but there is no 's' on the end of aqa-systemtest

Reply:
Thanks for spotting. Fixed!